kerchunk/nos-sqs-sensor.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Sensor
+metadata:
+  name: nos-sqs
+spec:
+  template:
+    serviceAccountName: operate-workflow-sa
+  dependencies:
+    - name: nos-dep
+      eventSourceName: nos-sqs
+      eventName: newofs
+  triggers:
+    - template:
+        name: nos-sqs-workflow
+        k8s:
+          operation: create
+          source:
+            resource:
+              apiVersion: argoproj.io/v1alpha1
+              kind: Workflow
+              metadata:
+                generateName: nos-sqs-workflow-
+              spec:
+                entrypoint: extract-sqs-message
+                arguments:
+                  parameters:
+                  - name: message
+                    # this is the value that should be overridden
+                    value: hello world
+                templates:
+                - name: extract-sqs-message
+                  inputs:
+                    parameters:
+                    - name: message
+                  script:
+                    image: miannuccirps/kerchunk-nos:latest
+                    env:
+                    - name: AWS_ACCESS_KEY_ID
+                      valueFrom:
+                        secretKeyRef:
+                          name: aws-secret
+                          key: accesskey
+                    - name: AWS_SECRET_ACCESS_KEY
+                      valueFrom:
+                        secretKeyRef:
+                          name: aws-secret
+                          key: secretkey
+                    command: [python]
```

Comment:
I hope we can move this Python code out of yaml!

Reply:
I can try to pass the parameter in as a string but out of the box this allows the template to use it as a native python dictionary.